packages/react/src/__tests__/__snapshots__/TextInput.test.tsx.snap

```diff
@@ -4173,6 +4185,14 @@ exports[`TextInput renders with a loading indicator 1`] = `
 
 }
 
+@media screen and (min-width:544px) {
```

Comment:
I wonder why these were added? 🤔

Reply:
Yeah, seemed strange but noticed they [already existed](https://github.com/primer/react/pull/5243/files#diff-5fcb2e729cbdf5bfdeb9af64ed597a2bd528d72fc44b4669a7316d7a3d741d69R4196-R4198) and so I figured it was some artifact of using styled components.